.travis.yml

```diff
@@ -46,6 +46,7 @@ requires:
     - intltool
     - libgsf
```

Comment:
Besides adding minizip, libgsf should be removed from the dependencies for all these builds just to check that it works without it.